index.html

```diff
@@ -305,7 +305,13 @@ <h2 class="tittle">Meet our experts</h2>
                   <img class="wow fadeInUp img-responsive img-thumbnail" src="http://res.cloudinary.com/dews/image/upload/c_scale,w_235/v1469466059/Clinic/dr.ankur.jpg">
                   <div class="team-address">
                     <p>Dr. Ankur Mishra</p>
-                    <span>BHMS, PGDCC</span>
+                    <span>BHMS, FHD, PGDCC (Dermatologist)</span>
+                  </div>
+                </li>
+                <li>
+                  <img class="wow fadeInUp img-responsive img-thumbnail" src="assets\images\IMG-20191201-WA0016.jpg">
```

Comment:
Maybe use https://squoosh.app/ to compress the image before uploading. To make the page faster.